README.md

```diff
@@ -191,22 +191,38 @@ unicli passwd -p pass -o output_file.pdf input_file.pdf new_owner_pass new_user_
 
 #### Optimize
 
-Optimize PDF files by removing redundant objects. The quality of the
-contained images can also be configured.
+Optimize PDF files by removing redundant objects.
```

Comment:
Optimize PDF files by optimizing structure, compression and image quality.
might be a better title.